components/blogger/actions/create-a-post/create-a-post.mjs

```diff
@@ -0,0 +1,54 @@
+import bloggerApp from "../../blogger.app.mjs";
+
+export default {
+  name: "Create a Post",
+  description: "Creates and publishes a new post or creates a new post as a draft. [See the docs here](https://developers.google.com/blogger/docs/3.0/reference/posts/insert).",
+  key: "blogger-create-post",
```

Comment:
Either the file should be named `create-post.mjs` or the `key` should be `blogger-create-a-post`;
```suggestion
  key: "blogger-create-a-post",
```